src/HeaderRow.tsx

```diff
@@ -10,9 +10,8 @@ type SharedDataGridProps<R, K extends keyof R, SR> = Pick<DataGridProps<R, K, SR
   | 'onSelectedRowsChange'
   | 'sortColumn'
   | 'sortDirection'
-  | 'onSort'
   | 'rowKey'
->;
+> & Required<Pick<DataGridProps<R, K, SR>, 'onSort'>>;
```

Comment:
In `SortableHeaderCell` we can now remove the `if (!onSort) return;` line.

Reply:
Done